content/docs/guides/components/tftraining.md

```diff
@@ -204,70 +204,6 @@ Typically you will want to change the following values
 
 1. Attach PV's if you want to use PVs for storage.
 
-### Submitting a TensorFlow training job
-
-**Note:** Before submitting a training job, you should have [deployed kubeflow to your cluster](#deploy-kubeflow). Doing so ensures that
-the [`TFJob` custom resource](https://github.com/kubeflow/tf-operator) is available when you submit the training job.
-
-We treat each TensorFlow job as a [component](https://ksonnet.io/docs/tutorial#2-generate-and-deploy-an-app-component) in your APP.
-
-### Run the TfCnn example
-
-Kubeflow ships with a [ksonnet prototype](https://ksonnet.io/docs/concepts#prototype) suitable for running the [TensorFlow CNN Benchmarks](https://github.com/tensorflow/benchmarks/tree/master/scripts/tf_cnn_benchmarks).
-
-You can also use this prototype to generate a component which you can then customize for your jobs.
-
-Create the component
-
-```
-CNN_JOB_NAME=mycnnjob
-
-ks registry add kubeflow-git github.com/kubeflow/kubeflow/tree/${VERSION}/kubeflow
```

Comment:
Where else in the docs is this information provided?

Reply:
this information is already there started from line 143, whereas this part from line 207 is repeated.